README.md

```diff
@@ -54,9 +54,23 @@ javafx {
 }
 ```
 
+## System Configuration
+
+### Mac OS
+
+To let TestFX manage Mouse and Keyboard, you have to authorized it.
+
+* Go to System Setting > Security and Privacy > Accessibility
+    * If you want to exectute the tests from your IDE, check your IDE in the list
```

Comment:
Typo: exectute: execute

I would rewrite as:

If you want to execute tests from your IDE make sure your IDE is selected.